src/main/java/org/mineacademy/boss/model/BossPlaceholders.java

```diff
@@ -191,7 +194,7 @@ else if ("respawn".equals(secondArg)) {
 			}
 		}
 
-		if ("name".equals(params) || "alias".equals(params) || "health".equals(params) || "top_damager".equals(params) || "top_damage".equals(params) || "location".equals(params) || "location_x".equals(params) || "location_y".equals(params) || "location_z".equals(params) || "location_world".equals(params)) {
+		if ("name".equals(params) || "alias".equals(params) || "health".equals(params) || params.startsWith("top_damager") || params.startsWith("top_damage") || "location".equals(params) || "location_x".equals(params) || "location_y".equals(params) || "location_z".equals(params) || "location_world".equals(params)) {
```

Comment:
It doesnt make sense to catch both params.startsWith("top_damager") || params.startsWith("top_damage") because  params.startsWith("top_damage") will catch params.startsWith("top_damager") too.